src/main/java/controller/LadderGameController.java

```diff
@@ -0,0 +1,65 @@
+package controller;
+
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+import model.Height;
+import model.Ladder;
+import model.LadderGenerator;
+import model.Players;
+import model.Prizes;
+import model.Width;
```

Comment:
모델의 거의 모든 파일을 import 하고 있으니 이런식으로 바꿔도 좋을 것 같아요!
```suggestion
import model.*;
```